agent/util-scripts/pcp-mapping.json

```diff
@@ -0,0 +1,382 @@
+            "docker.memory_stats.stats.total_active_anon",
+            "docker.memory_stats.stats.total_active_file",
+            "docker.memory_stats.stats.total_cache",
+            "docker.memory_stats.stats.total_dirty",
+            "docker.memory_stats.stats.total_inactive_anon",
+            "docker.memory_stats.stats.total_inactive_file",
+            "docker.memory_stats.stats.total_mapped_file",
+            "docker.memory_stats.stats.total_pgfault",
+            "docker.memory_stats.stats.total_pgmajfault",
+            "docker.memory_stats.stats.total_pgpgin",
+            "docker.memory_stats.stats.total_pgpgout",
+            "docker.memory_stats.stats.total_rss",
+            "docker.memory_stats.stats.total_rss_huge",
+            "docker.memory_stats.stats.total_swap",
+            "docker.memory_stats.stats.total_unevictable",
+            "docker.memory_stats.stats.total_writeback",
+            "docker.memory_stats.stats.unevictable",
+            "docker.memory_stats.stats.writeback",
+            "docker.control.timing"
+        ],
+        "probe": "docker.version"
+    },
+    "iostat": {
+        "ident": "used for monitoring system input/output device",
+        "metrics": [
+            "disk.dev.read_bytes",
+            "disk.dm.read_bytes",
+            "disk.dev.write_bytes",
+            "disk.dm.write_bytes",
+            "kernel.all.cpu.user",
+            "kernel.all.cpu.nice",
+            "kernel.all.cpu.sys",
+            "kernel.all.cpu.wait.total",
+            "kernel.all.cpu.steal",
+            "kernel.all.cpu.idle",
+            "disk.all.read",
+            "disk.all.read_merge",
+            "disk.all.blkread",
+            "disk.all.read_rawactive",
+            "disk.all.write",
+            "disk.all.write_merge",
+            "disk.all.blkwrite",
+            "disk.all.write_rawactive",
+            "disk.all.avactive",
+            "disk.dev.read",
+            "disk.dm.read",
+            "disk.dev.read_merge",
+            "disk.dm.read_merge",
+            "disk.dev.blkread",
+            "disk.dm.blkread",
+            "disk.dev.read_rawactive",
+            "disk.dm.read_rawactive",
+            "disk.dev.write",
+            "disk.dm.write",
+            "disk.dev.write_merge",
+            "disk.dm.write_merge",
+            "disk.dev.blkwrite",
+            "disk.dm.blkwrite",
+            "disk.dev.write_rawactive",
+            "disk.dm.write_rawactive",
+            "disk.all.read_merge",
+            "disk.all.write_merge",
+            "disk.all.read",
+            "disk.all.write",
+            "disk.all.read_bytes",
+            "disk.all.write_bytes",
+            "disk.dev.avactive",
+            "disk.dm.avactive",
+            "disk.all.read_rawactive",
+            "disk.all.write_rawactive"
+        ],
+        "probe": "kernel.percpu.cpu"
+    },
+    "kvmstat": {
+        "ident": "metrics used by the pcp-kvm(1) command        ",
```

Comment:
This has trailing whitespace in the quotes, which is weird. Is that really deliberate?